build.sbt

```diff
@@ -1,10 +1,29 @@
+import sbt.Keys.libraryDependencies
 
-crossScalaVersions := Seq("2.11.12", "2.12.12")
+lazy val CommonSettings = Seq(
+  version := (
+    if (git.gitCurrentBranch.value == "main") {
+      "latest"
+    } else {
+      git.gitCurrentBranch.value
+    }) + "-SNAPSHOT",
+)
 
-organization := "co.datamechanics"
-name := "delight"
-version := "latest-SNAPSHOT"
+lazy val global = project
```

Comment:
nit: a [convention](https://www.scala-sbt.org/1.x/docs/Multi-Project.html#Aggregation) is to use `root`